spec/lib/binary-buffer-module.spec.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+
+describe("binaryBuffer", () => {
+    var binaryBuffer, copy;
+
+    beforeEach(() => {
+        binaryBuffer = require("../../lib/binary-buffer")();
+    });
+    [
+        "some string",
+        "",
+        new Buffer("some buffer"),
+        123
+    ].forEach((scenario) => {
+        describe(".toBuffer()", () => {
+            it("should convert a string to a buffer", () => {
+                scenario = binaryBuffer.toBuffer(scenario);
+                expect(scenario).toEqual(jasmine.any(Buffer));
+            });
+            it("should not alter the stringOrBuffer if it is already a buffer", () => {
+                copy = scenario;
+                scenario = binaryBuffer.toBuffer(scenario);
+                expect(scenario).toEqual(jasmine.any(Buffer));
+                expect(scenario).toBe(copy);
```

Comment:
Not pushed.

Also, I would expect tests to be formatted like this:

    [
        "some string",
        new Buffer("some buffer")
    ].forEach((input) => {
        describe(".toBuffer()", () => {
            it("makes a buffer", () => {
                expect(Buffer.isBuffer(binaryBuffer.toBuffer(input))).toBe(true);
            });
            it("has the right content", () => {
                expect(binaryBuffer.toBuffer(input).toString()).toEqual(input.toString());
            });
        });
        describe(".toString(), () => {
            // test returned type
            // test returned value when converted to a string
        });
    });

This is super close to what you are currently doing, but the second test is worded badly.  You say something like "it should not convert the type" instead of "the value is correct".